apps/babelstream/package.py

```diff
@@ -0,0 +1,363 @@
+
+
+
+# raise InstallError('This package does not build on Mac OS X!')
+
+    #==> Error: variant "cuda" not found in package "babelstream" [required from package "babelstream"] this line below fixes this
+    #variant("cuda", default=False, description="Enable CUDA support")
+    #variant("cuda_arch", values=str, default="none", description="The CUDA architecture being used.")
+    
+
+
+    variant("sycl", default=False, description="Enable SYCL support")
+    
+
+    #Defined in CudaPackage
+    #variant("cuda", default=False, description="Enable CUDA support")
+    variant("sycl2020", default=False, description="Enable SYCL support")
+    variant("omp", default=False, description="Enable OpenMP support")
+    variant("ocl", default=False, description="Enable OpenCL support") ## Crushes llvm requires RAM
+    variant("tbb", default=False, description="Enable TBB support")
+    variant("acc", default=False, description="Enable OpenACC support") 
+    variant("thrust", default=False, description="Enable THRUST support")
+    variant("raja", default=False, description="Enable RAJA support") 
+
+    variant("stddata", default=False, description="Enable STD-data support")
+    variant("stdindices", default=False, description="Enable STD-data support")
+    variant("stdranges", default=False, description="Enable STD-data support")
+    
+    variant("dir", values=str, default="none", description="Enable Directory support")
+
+    conflicts("dir=none",when="+kokkos",msg="KOKKKOS requires architecture to be specfied by dir=")
+    variant("kokkos", default=False, description="Enable KOKKOS support")
+    
+    #ACC conflict
+    variant("cpu_arch", values=str, default="none", description="Enable CPU Target for ACC")
+    # conflicts("cpu_arch=none",when="+acc",msg="KOKKKOS requires architecture to be specfied by dir=")
+    variant("target", values=str, default="none", description="Enable CPU Target for ACC")
+    
+    
+    # CUDA conflict 
+    # this is not provided in CudaPackage
+    conflicts("cuda_arch=none",when="+cuda",msg="CUDA requires architecture to be specfied by cuda_arch=")
+    variant("mem", values=str, default="DEFAULT", description="Enable MEM Target for CUDA")
+        
+    #download raja from https://github.com/LLNL/RAJA
+    conflicts("dir=none",when="+raja",msg="KOKKKOS requires architecture to be specfied by dir=")
+
+    #Thrust Conflict
+    conflicts("~cuda", when="+thrust", msg="Thrust requires +cuda variant")
+
+    #TBB Dependency
+    depends_on("intel-oneapi-tbb", when="+tbb")
+
+    #depends_on("cmake@3.8", when="+stdindices")
+    depends_on("cmake@3.23.2", when="+stddata")
+    depends_on("gcc@12.1.0", when="+stddata")
+
+    depends_on("cmake@3.23.2", when="+stdindices")
+    depends_on("gcc@12.1.0", when="+stdindices")
+    
+    depends_on("cmake@3.23.2", when="+stdranges")
+    depends_on("gcc@12.1.0", when="+stdranges")
+    
+    #OMP Dependency
+    depends_on("openmpi", when="+omp")
```

Comment:
`openmpi` isn't OpenMP, am I missing something else?

Reply:
It was from the original PR code from [here](https://github.com/spack/spack/pull/32680/files#diff-017e0b85946d9e442915b5b2435f385c86545eeb3fa11c17f7b083fff9a64dafR73) but OpenMP and openMPI are different and actually there is no package for `openMP`. so we could delete this line it is irrelevant and actually we don't need any extra dependency for openMP. Thanks for pointing this out